package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "gaia-icons",
-  "version": "0.9.2",
+  "version": "0.9.3",
```

Comment:
This would be a **major** change, so I’d suggest using `0.10.0` for the version number.